lib/core/test/unit/Cardano/Byron/Codec/CborSpec.hs

```diff
@@ -67,7 +67,7 @@ import qualified Data.ByteString.Lazy as BL
 {- HLINT ignore spec "Use head" -}
 
 spec :: Spec
-spec = do
+spec = parallel $ do
```

Comment:
Is it possible to have global parallel setting for the unit test-suite?

Reply:
There a few tests we don't run in parallel, like those starting local servers and playing around with ports. Databases were also excluded initially but I've found a way to have them running in parallel too. 

Arguably, we could move the problematic tests in a another test suite like `integration`. I am also not sure at this stage whether we can have a single global `parallel`, maybe by tweaking:

https://github.com/input-output-hk/cardano-wallet/blob/4df2ecc5d5259ce333de6b2dfa39b6ac51b17652/lib/core/test/unit/Main.hs#L9-L10